test/instances/eeth/EETHHyperdrive.t.sol

```diff
@@ -0,0 +1,618 @@
+            hyperdriveBalancesBefore
+        );
+    }
+
+    /// Short ///
+
+    function test_round_trip_short_instantaneous(uint256 shortAmount) external {
+        // Bob opens a short with base.
+        shortAmount = shortAmount.normalizeToRange(
+            2 * hyperdrive.getPoolConfig().minimumTransactionAmount,
+            hyperdrive.calculateMaxShort()
+        );
+        (uint256 maturityTime, uint256 basePaid) = openShort(bob, shortAmount);
+
+        // Get some balance information before the withdrawal.
+        (
+            uint256 totalSupplyAssetsBefore,
+            uint256 totalSupplySharesBefore
+        ) = getSupply();
+        AccountBalances memory bobBalancesBefore = getAccountBalances(bob);
+        AccountBalances memory hyperdriveBalancesBefore = getAccountBalances(
+            address(hyperdrive)
+        );
+
+        // Bob closes his long with base as the target asset.
+        uint256 baseProceeds = convertToBase(
+            closeShort(bob, maturityTime, shortAmount, false)
+        );
+
+        // Bob should receive approximately as much base as he paid since no
+        // time as passed and the fees are zero.
+        assertApproxEqAbs(baseProceeds, basePaid, 1e9);
+
+        // Ensure that the withdrawal was processed as expected.
+        verifyWithdrawal(
+            bob,
+            baseProceeds,
+            false,
+            totalSupplyAssetsBefore,
+            totalSupplySharesBefore,
+            bobBalancesBefore,
+            hyperdriveBalancesBefore
+        );
+    }
+
+    function test_round_trip_short_maturity(
+        uint256 shortAmount,
+        uint256 variableRate
+    ) external {
+        // Bob opens a short with base.
+        shortAmount = shortAmount.normalizeToRange(
+            2 * hyperdrive.getPoolConfig().minimumTransactionAmount,
+            hyperdrive.calculateMaxShort()
+        );
+        (uint256 maturityTime, ) = openShort(bob, shortAmount);
+
+        // The term passes and some interest accrues.
+        variableRate = variableRate.normalizeToRange(0, 2.5e18);
+        advanceTime(POSITION_DURATION, int256(variableRate));
+
+        // Get some balance information before the withdrawal.
+        (
+            uint256 totalSupplyAssetsBefore,
+            uint256 totalSupplySharesBefore
+        ) = getSupply();
+        AccountBalances memory bobBalancesBefore = getAccountBalances(bob);
+        AccountBalances memory hyperdriveBalancesBefore = getAccountBalances(
+            address(hyperdrive)
+        );
+
+        // Bob closes his long with base as the target asset.
+        uint256 baseProceeds = convertToBase(
+            closeShort(bob, maturityTime, shortAmount, false)
+        );
+
+        // Bob should receive almost exactly the interest that accrued on the
+        // bonds that were shorted.
+        assertApproxEqAbs(
+            baseProceeds,
+            shortAmount.mulDown(uint256(variableRate)),
+            1e5
```

Comment:
Same here. Might not be effected by the conversions library changes, but it's worth a double-check

Reply:
aye